src/pages/Faucet/Faucet.tsx

```diff
@@ -2,14 +2,18 @@ import { Button, ModalContainer, PrivateKeyCheckWrapper } from 'components';
 import { DataTestIdsEnum } from 'localConstants';
 import { FaucetModal } from './components/FaucetModal';
 import { useModal } from '../../hooks';
+import { useSelector } from 'react-redux';
+import { networkSelector } from 'redux/selectors';
```

Comment:
Maybe we could order and group these imports? Package imports should be grouped first, then absolute imports, and then relative imports, in that order. 👍